### ISSUE TITLE
System tray opens at the center of desktop In Wayland session

### ISSUE THREAD

**Issue by Sarcasmo220** [nx-desktop/nx-systemtray-applet#9]
**Describe the bug**
In Wayland session, when clicking on an applet in the system tray, it opens in the center of the desktop instead of the side the system tray is located. This behavior began after updating the system on April 4th 2021

**To Reproduce**
1. Login under Wayland session
2. Click on a applet icon in the system tray
3. See error

**Expected behavior**
Applet opens on the side of the desktop where the system tray is located. Expected behavior is observed in X11 session.

**Screenshots**
![Screenshot_20210412_211012](https://user-images.githubusercontent.com/68247163/114496473-e0b0fe80-9bd4-11eb-9f1c-4e0689671873.png)

**Desktop (please complete the following information):**
 - OS: Nitrux
 - Version 1.3.9 build.260321
 - Session: Wayland

**Comment by UriHerrera** [comment]
That behavior may be caused by Wayland itself or KWin Wayland and not necessarily because of the applet. Do you have multiple monitors connected?.

**Comment by Sarcasmo220** [comment]
No, it is the single laptop display. I have not tested Wayland sessions on other KDE distros but that could be a way to see if other system trays have the same behavior.
